src/stirling/source_connectors/socket_tracer/uprobe_manager.cc

```diff
@@ -415,7 +409,7 @@ StatusOr<int> UProbeManager::AttachOpenSSLUProbesOnDynamicLib(uint32_t pid) {
     // before the BPF map is updated. This value is cleaned up when the upid is
     // terminated, so if attachment fails it will be deleted prior to the pid being
     // reused.
-    openssl_source_map_->UpdateValue(pid, ssl_source);
+    PX_RETURN_IF_ERROR(openssl_source_map_->SetValue(pid, ssl_source));
```

Comment:
I don't think this should cause a uprobe attachment to fail. Even though it's ideal to have each traced application attributed correctly, the TLS tracing functions properly if this fails (falling back to `kSSLUnspecified`). Given that my preference is to not check this return value for an error.